aiida_restapi/routers/daemon.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""Declaration of FastAPI router for daemon endpoints."""
+from __future__ import annotations
+
+import typing as t
+
+from aiida.cmdline.utils.decorators import with_dbenv
+from aiida.engine.daemon.client import DaemonException, get_daemon_client
+from fastapi import APIRouter, Depends, HTTPException
+from pydantic import BaseModel, Field
+
+from ..models import User
+from .auth import get_current_active_user
+
+router = APIRouter()
+
+
+class DaemonStatusModel(BaseModel):
```

Comment:
Should this be moved to `aiida_restapi.models`?

Reply:
Don't know. Kept it here for now since it doesn't seem to be a "database" model as the others, but more a proxy model for a response data structure. Happy to move it there, if that is the preferred organization for this application.